src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -64,6 +65,11 @@ private void fillInputFieldWithMilestoneName(String milestoneName) {
         inputField.setText(milestoneName);
     }
 
+    private final void positionDialog(Stage stage) {
```

Comment:
This (together with other similar functionalities) seems to be extractable into a separate class that the pickers extend from. But I guess this should do for now.